frontend/src/container/EditAlertChannels/index.tsx

```diff
@@ -389,24 +399,35 @@ function EditAlertChannels({
 				}
 
 				if (response && response.statusCode === 200) {
+					testSuccess = true;
 					notifications.success({
 						message: 'Success',
 						description: t('channel_test_done'),
 					});
 				} else {
+					testSuccess = false;
 					notifications.error({
 						message: 'Error',
 						description: t('channel_test_failed'),
 					});
 				}
 			} catch (error) {
+				testSuccess = false;
```

Comment:
is this variable needed ?

Reply:
yes, majorly because of 2 reasons - 
- response is present in inner block scope
- success will captured by response status, but error are coming from different areas, like default case of switch etc.